src/level/robotmain.cpp

```diff
@@ -778,6 +779,26 @@ bool CRobotMain::ProcessEvent(Event &event)
     }
 
     if (event.type == EVENT_KEY_DOWN &&
+        event.GetData<KeyEventData>()->key == KEY(UP) && m_cmdEdit)
+    {
+        Ui::CEdit* pe = static_cast<Ui::CEdit*>(m_interface->SearchControl(EVENT_CMD));
+        if (pe == nullptr) return false;
+        std::string cmd = GetNextFromCommandHistory();
+        if (cmd != "") pe->SetText(cmd);
```

Comment:
It would be nice to change this to `if (!cmd.empty())` for readability (and possibly performance but that's not that big of a deal here)